casepro/msgs/migrations/0059_new_modified_msgs_index.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.7 on 2017-11-20 19:32
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+SQL = """
+CREATE INDEX CONCURRENTLY msgs_message_org_modified_on
+ON msgs_message(org_id, has_labels, modified_on, created_on DESC)
+WHERE is_active = TRUE AND is_handled = TRUE;
+"""
```

Comment:
Adding a better index here for fetching modified messages

Reply:
Is there a similar index that can now be dropped?

Reply:
There's a Django managed index that gets dropped after this is created by setting index_together to none